### PR TITLE
[Manta] CI runtime upgrade test for manta parachain

### LINKED ISSUE #261: Runtime upgrade test with `manta-testnet-ci.json`
https://github.com/Manta-Network/Manta/blob/manta/.github/workflows/publish-draft-releases.yml#L290-L298
the manta variant in the matrix doesn't have a chain-spec defined
First we need to make a genesis with public sessions keys like https://github.com/Manta-Network/Manta/blob/manta/genesis/calamari-testnet-ci-genesis.json
Needed because we can just expose their secrets in the CI without worry.
Just need a function in `chain_spec.rs` to create the json file, then delete the function and just keep the json
Then need to integrate it in the CI, like the runtime upgrade test for `calamari-testnet-ci`

### PR DESCRIPTION
**Pull request by ghzlatarev**
## Description

closes: #261

* Add `manta-testnet-ci-genesis.json` for CI runtime upgrade test for manta parachain. It uses the same secrets as the `calamari-testnet-ci-genesis.json`
* Add missing matrix argument in CI runtime upgrade test to refer to this new chain-spec.

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (`manta` or `dolphin`) with right title (start with [Manta] or [Dolphin]),
- [x] Linked to Github issue with discussion and accepted design OR have an explanation in the PR that describes this work.
- [x] Wrote unit tests.
- [x] Updated relevant documentation in the code.
- [x] Re-reviewed `Files changed` in the Github PR explorer.
- [x] If runtime changes, need to update the version numbers properly:
   * `authoring_version`: The version of the authorship interface. An authoring node will not attempt to author blocks unless this is equal to its native runtime.
   * `spec_version`: The version of the runtime specification. A full node will not attempt to use its native runtime in substitute for the on-chain Wasm runtime unless all of spec_name, spec_version, and authoring_version are the same between Wasm and native.
   * `impl_version`: The version of the implementation of the specification. Nodes are free to ignore this; it serves only as an indication that the code is different; as long as the other two versions are the same then while the actual code may be different, it is nonetheless required to do the same thing. Non-consensus-breaking optimizations are about the only changes that could be made which would result in only the impl_version changing.
   * `transaction_version`: The version of the extrinsics interface. This number must be updated in the following circumstances: extrinsic parameters (number, order, or types) have been changed; extrinsics or pallets have been removed; or the pallet order in the construct_runtime! macro or extrinsic order in a pallet has been changed. If this number is updated, then the spec_version must also be updated 
- [x] If needed, notify the committer this is a draft-release and a tag is needed after merging the PR.
- [x] Verify benchmarks & weights have been updated for any modified runtime logics

### CODE REVIEW

File: .github/workflows/generate_calamari_weights_files.yml [hunk 1]
```diff
@@ -176,6 +176,14 @@ jobs:
...
               name: pallet_multisig
             iterations: 20
+          -
+            extrinsic:
+              id: '*'
+              name: pallet_timestamp
+            pallet:
+              id: pallet_timestamp
+              name: pallet_timestamp
+            iterations: 20
```
Comment (ghzlatarev) [inline .github/workflows/generate_calamari_weights_files.yml:202]: I must've deleted this earlier by accident.

**Comment by stechu** [comment]
merge conflicts?

**Comment by ghzlatarev** [comment]
> merge conflicts?

@stechu Fixed the conflicts.

**Review by Dengjianping — APPROVED** [review]
LGTM!

File: .github/workflows/generate_calamari_weights_files.yml [hunk 2]
```diff
@@ -192,6 +192,14 @@ jobs:
...
               name: pallet_utility
             iterations: 20
+          -
+            extrinsic:
+              id: '*'
+              name: pallet_timestamp
+            pallet:
+              id: pallet_timestamp
+              name: pallet_timestamp
+            iterations: 20
```
Comment (stechu) [inline .github/workflows/generate_calamari_weights_files.yml:202]: did we miss this pallet before?

**Review by grenade — APPROVED** [review]

**Review by stechu — APPROVED** [review]

File: .github/workflows/generate_calamari_weights_files.yml (hunk 2 above)
Comment (stechu) [reply .github/workflows/generate_calamari_weights_files.yml:202]: @ghzlatarev

**Pull request merged**